### PR TITLE
Fix dependency installation on Google Colab for `cratedb-vectorstore-rag-openai-sql.ipynb`

### PR DESCRIPTION
**Pull request by hammerhead**
## Summary of the changes / Why this is an improvement

Google Colab failed to resolve dependencies, see #255.

## Checklist

 - [X] Link to issue this PR refers to (if applicable): 
 - [X] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb [hunk 1]
```diff
@@ -426,7 +431,7 @@
     "    ]\n",
     ")\n",
     "\n",
-    "chat_completion.choices[0].message.content"
+    "print(chat_completion.choices[0].message.content)"
```
Comment (hammerhead) [inline topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb:434]: Slightly nicer output with the line breaks that `print` adds.

File: topic/machine-learning/llm-langchain/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Real.
 crash
-crate[sqlalchemy]
+crate[sqlalchemy]>=0.34,<0.35
```
Comment (hammerhead) [inline topic/machine-learning/llm-langchain/requirements.txt:3]: Google Colab was unable to resolve the dependency, although there was no actual conflict with how `crate[sqlalchemy]` is required in `langchain[cratedb']` and `cratedb-toolkit`. Adding an explicit constraint helped.

File: topic/timeseries/explore/timeseries-queries-and-visualization.ipynb [hunk 3]
```diff
@@ -54,7 +54,10 @@
     "## Install required dependencies\n",
     "\n",
     "*Note on Google Colab:* Google Colab currently uses pandas 1.5.3. During the installation\n",
-    "of the dependencies, you will see a compatibility error, which can be ignored."
+    "of the dependencies, you will see a compatibility error, which can be ignored.\n",
+    "\n",
+    "[CrateDB Cloud]: https://console.cratedb.cloud\n",
+    "[Time-Series: Analyzing Weather Data]: ???"
```
Comment (hammerhead) [inline topic/timeseries/explore/timeseries-queries-and-visualization.ipynb:60]: Not sure what this is supposed to link to. Anyone knows?

File: topic/timeseries/explore/timeseries-queries-and-visualization.ipynb [hunk 4]
```diff
@@ -489,8 +492,8 @@
     "and lowest temperature for each place, and when the measurement happened.\n",
     "\n",
     "In this case, simple groupings are not enough. CrateDB provides enhanced tools\n",
-    "for time series data. You can use the `max_by(returned_value, maximized_value)`\n",
-    "or `min_by(returned_value, maximized_value)` functions, which return a value\n",
+    "for time series data. You can use the `MAX_BY(returned_value, maximized_value)`\n",
+    "or `MIN_BY(returned_value, maximized_value)` functions, which return a value\n",
```
Comment (hammerhead) [inline topic/timeseries/explore/timeseries-queries-and-visualization.ipynb:496]: For consistency, it's used in uppercase in the corresponding query.

File: topic/timeseries/explore/timeseries-queries-and-visualization.ipynb (hunk 3 above)
Comment (ckurze) [reply topic/timeseries/explore/timeseries-queries-and-visualization.ipynb:60]: Let's link it here: https://cratedb.com/docs/cloud/en/latest/tutorials/time-series.html#time-series

File: topic/machine-learning/llm-langchain/requirements.txt (hunk 2 above)
Comment (amotl) [reply topic/machine-learning/llm-langchain/requirements.txt:3]: There is a constant churn of mutual dependency updates required to balance code moving around. Thank you for figuring out how to address this particular woe.

**Review by amotl — APPROVED** [review]
Thanks for fixing the dependencies, and for copy-editing.

File: topic/machine-learning/llm-langchain/requirements.txt (hunk 2 above)
Comment (amotl) [reply topic/machine-learning/llm-langchain/requirements.txt:3]: Let me know if you think there are more serious issues here, like if there may be something wrong with the recent 0.35.x releases of https://pypi.org/project/crate/.

**Pull request merged**